roles/lib_openshift/src/lib/volume.py

```diff
@@ -0,0 +1,36 @@
+# pylint: skip-file
+
+class Volume(object):
```

Comment:
Is there a standalone module for volume? should it be a pre-req to this PR instead of bringing in just a subset of it?